### ISSUE TITLE
Next up for this project: add option to the simulation 

### ISSUE THREAD

**Issue by Kenkleinman** [katieleap/normanal#7]
Allow a) specific values for the persons/cluster or b) distributions (NB, Poisson, integer uniform)

**Comment by katieleap** [comment]
The specific values for the persons/cluster are currently input from the sidebar.

**Comment by katieleap** [comment]
Never mind, I'm thinking of a different code chunk.

**Comment by katieleap** [comment]
Current input variables:
effect.size=as.numeric(input$d)
n.clusters=as.numeric(input$N),
cluster.size=as.numeric(input$M),
btw.clust.var=as.numeric(input$sigmab), 
indiv.var=as.numeric(input$sigma),

Would the distribution selection be a radio button or a drop-down list?

**Comment by Kenkleinman** [comment]
Yes, this would be an add-on: we would draw values from one of the distributions I mentioned, then plug them into _cluster.size_.  If the user chooses to input exact values, we can let them input on the app, or upload a file, I guess.

**Comment by katieleap** [comment]
So the sim is still normal? But the clusters aren't?

**Comment by Kenkleinman** [comment]
The outcome is still normal, but the distribution of the number of people in each cluster is not.

**Comment by Kenkleinman** [comment]
Another thing for "later".
